include/openenclave/corelibc/stdlib.h

```diff
@@ -52,6 +52,8 @@ void* oe_memalign(size_t alignment, size_t size);
 
 int oe_posix_memalign(void** memptr, size_t alignment, size_t size);
 
+void* oe_aligned_alloc(size_t alignment, size_t size);
```

Comment:
Can you double check how do we make this symbol with DEFAULT visibility?

Reply:
Also can you revisit the list of symbols that the symcrypt module depend on and update https://github.com/openenclave/openenclave/blob/master/tests/module_loading/module/module.c accordingly?